website/common/script/content/faq.js

```diff
@@ -14,7 +14,11 @@ for (let i = 0; i <= NUMBER_OF_QUESTIONS; i++) {
   let question = {
     question: t(`faqQuestion${i}`),
     ios: t(`iosFaqAnswer${i}`),
-    web: t(`webFaqAnswer${i}`),
+    web: t(`webFaqAnswer${i}`, {
+      //  TODO: Need to pull these values from nconf
```

Comment:
That's the goal, but given that these files are shared between the client and the server, I'm having a hard time figuring out a single way to access the env variables.

process.env doesn't have them.
env is not defined.
nconf is for server side only.

Seems like I need to pass these in when loading common... My ask was to get this checked in as is as it adds value and then proceed to figure out how to handle these.